pandas/tests/test_nanops.py

```diff
@@ -1015,3 +1017,25 @@ def test_use_bottleneck():
         assert not pd.get_option('use_bottleneck')
 
         pd.set_option('use_bottleneck', use_bn)
+
+
+@pytest.mark.parametrize("numpy_op, passes_all_numpy_version, expected", [
+    (np.sum, True, 10),
+    (np.nansum, True, 10),
+    (np.mean, True, 2.5),
+    (np.nanmean, True, 2.5),
+    (np.median, True, 2.5),
+    (np.nanmedian, True, 2.5),
+    (np.min, True, 1),
+    (np.nanmin, False, 1),
+    (np.max, True, 4),
+    (np.nanmax, False, 4),
+])
+def test_numpy_ops(numpy_op, passes_all_numpy_version, expected):
+    # GH8383
```

Comment:
you can just make 2 tests
1 for numpy >= 1.13 and 1 for < 1.13 which assert those functions which exist there